content/docs/contributing/to-docs/style-guide/quick-reference.md

```diff
@@ -0,0 +1,32 @@
+| [Idioms, slang, colloquialisms, jokes, or jargon](guidance.md#idioms-slang-colloquialisms-jokes-or-jargon) | Don't use. These words and phrases may be understood by native U.S. English speakers, but are difficult to translate.  |
+| **Formatting** |
+| [Hugo](../hugo.md) and [Markdown](format.md) | Documentation is written in Markdown (`.md`) files and built by [Hugo](https://gohugo.io/), a static site generator. It's primarily formatted using [Markdown syntax](https://www.markdownguide.org/basic-syntax/), with support for in-line HTML. Additional content formatting includes images, videos, and callout boxes. Callout boxes are implemented through [shortcodes](shortcodes.md). |
+| [Metadata](metadata.md) | Documentation must comply with metadata requirements, including *sectiontitle*, *title*, *description*, and *weight*. |
+| [Topic headings](format.md#topic-headings) | Don't use H1 (`#`) heading; instead, topic title is defined by `title` in the metadata. Section titles must be an H2 (`##`) heading. Subsection titles follow as H3 (`###`) and H4 (`####`) headings.  |
+| [User interface, inputs, and hotkeys](format.md#user-interface-inputs-and-hotkeys) | Bold all instances. |
+| [Files, directories, and paths](format.md#files-directories-and-paths) | Use code-style. Provide context on what the path is relative to. When applicable, link to the file in the [o3de repository](https://github.com/o3de/o3de).  |
+| [Applications, Tools, Gems, and components](format.md/#applications-tools-gems-and-components) | Bold only the first on-page appearance. Subsequent appearances on the same page remain unformatted.  |
+| [Code, commands, and API objects](format.md#code-commands-and-apis) | Use code-style for commands and API objects. Use code blocks for multiple lines of code. For API documentation, you may need to use advanced formatting -- see [[Formatting for API Documentation]]. |
+| [Terminology](format.md/#terminology) | Be consistent in the use of terms. Italicize new terms, and follow with definition. Terminology that's unique to O3DE must be included in the Terminology page.  |
+| [Trademark terminology](format.md/#trademark) | Properly format trademark titles and terminology according to its use from the source. Provide a link to the source's relevant material. |
+| **Additional content** |
+| [Images](media.md#adding-images-with-markdown-syntax) | Static images must be `.png` format. Diagrams must be `.svg` format. |
+| [Animated images](media.md#animated-images) | Use short videos (of a few seconds) to demonstrate a feature's functionality. Prefer to use `.mp4` files that are less than 512 KB. `.ogg` and `.webm` formats, and up to 1 MB of size are also accepted. `.gif` format is not permitted. |
```

Comment:
```suggestion
| [Animated images](media.md#animated-images) | Use short videos (of a few seconds) to demonstrate a feature's functionality. Prefer to use `.mp4` files that are less than 512 KB. `.ogg` and `.webm` formats up to 1 MB of size are also accepted. Use of `.gif` format is deprecated and no longer permitted. |
```

Reply:
I'm trying to capture two separate rules in a single sentence: (1) that `.ogg` and `.webm` formats are ok, and (2) that regardless of the format, videos cannot exceed 1 MB (though less than 512 KB is preferred). 
I think the suggestion doesn't capture that they are separate rules. 

What about the following?
```suggestion
| [Animated images](media.md#animated-images) | Use short videos (of a few seconds) to demonstrate a feature's functionality. Prefer to use `.mp4` files that are less than 512 KB. `.ogg` and `.webm` formats are also accepted. The video must not exceed 1 MB of size. Use of `.gif` format is deprecated and no longer permitted. |
```

Reply:
I agree with Mike's point and recommend his suggestion.